### DISCUSSION TITLE
Markdown rendering bug in raster_zonal_stats

### DISCUSSION THREAD

**Discussion by tm-jc-nacpil** [thinkingmachines/geowrangler#136] [Usability Testing Q&A] [answered]
Hi GW team! Flagging a minor issue I encountered in the documentation. The arguments list for [raster_zonal_stats](https://geowrangler.thinkingmachin.es/raster_zonal_stats.html) at located at the `https://geowrangler.thinkingmachin.es/` domain does not seem to render properly, making it hard to read (see screenshot below)

![image](https://user-images.githubusercontent.com/105769386/186404066-f5b80e68-13f7-4462-9331-eff4cc5846eb.png)

I saw a similar link https://geowrangler.web.app/raster_zonal_stats.html which renders this table properly however. Which one should we refer to moving forward?

**Comment by ghost** [comment]
@butchtm tagging you on this clarification/minor issue

**Reply by butchtm** [reply]
@tm-jc-nacpil  will look into this. something weird in the migration to the new doc site. 
@tm-kah-alforja consider tagging this as a documentation bug?

**Reply by ghost** [reply]
created issue #137 for this as well and added to our Usability Testing Results column

**Comment by butchtm** [comment, marked as answer]
Closing this as fixed in #137
